index.html

```diff
@@ -742,7 +742,10 @@ <h3>Validate Algorithm</h3>
         </li>
         <li>
 Let |status| be the value at the position indicated by the
-|credentialIndex| times the |size| in the |bitstring|.
+|credentialIndex| times the |size| in the |bitstring|. If the |credentialIndex|
+is a value outside of the range of the |bitstring|, a
+<a data-cite="VC-DATA-MODEL-2.0#RANGE_ERROR">RANGE_ERROR</a> MUST be
```

Comment:
The current VCDM does not specify a RANGE_ERROR. It only defined PARSING_ERROR, CRYPTOGRAPHIC_SECURITY_ERROR, and MALFORMED_VALUE_ERROR, see https://www.w3.org/TR/vc-data-model-2.0/#problem-details

Reply:
I've opened a PR to register this new type here: https://github.com/w3c/vc-data-model/pull/1405.